CONTRIBUTING.md

```diff
@@ -93,6 +85,12 @@ If your PR is contributing new functionality, did you create an issue to discuss
     - You can run the tests by executing `bundle install` and then `bundle exec rspec`.
 - Your code editor should indent using spaces with a tab size of 2 spaces.
 
+To submit the changes to the fastlane repo, you have to do the following:
+
+- Squash your commits into one. For example, to squash three commits into one, do the following: `$ git rebase -i HEAD~3`. In the text editor that comes up, replace the words "pick" with "squash" next to the commits you want to squash. Save and close the editor. For more information, take a look at [7.6 Git Tools - Rewriting History](http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)
```

Comment:
We can now do that with the new GitHub feature, right? https://github.com/blog/2141-squash-your-commits